service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/ManageLegalRepresentativesValidator.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.service.validators;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.util.Lists;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.LegalRepresentative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.OrganisationService;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ManageLegalRepresentativesValidator {
+
+    private final OrganisationService organisationService;
+
+    public List<String> validate(List<Element<LegalRepresentative>> legalRepresentatives) {
+        log.info("validate {}",legalRepresentatives);
+        return Optional.ofNullable(legalRepresentatives)
+            .orElse(Lists.emptyList()).stream().flatMap(
+                legalRepresentativeElement -> performBasicValidation(legalRepresentativeElement.getValue()).stream()
+        ).collect(Collectors.toList());
```

Comment:
maybe results should be unique? What if organisation name is not provided for 2 reps?  Should same message be displayed once or twice? There is not unit test that covers multi reps scenario

Reply:
done, now it gives the item number